tool/common/common.go

```diff
@@ -98,3 +101,34 @@ func ShowSessions(events []events.AuditEvent, format string, w io.Writer) error
 		return trace.BadParameter("unknown format %q", format)
 	}
 }
+
+// ShowClusterAlerts shows cluster alerts with the given label.
+func ShowClusterAlerts(ctx context.Context, client local.ClusterAlertGetter, w io.Writer, labels map[string]string, ignore map[string]string) error {
+	// get any "on login" alerts
+	alertCtx, _ := context.WithTimeout(ctx, constants.TimeoutGetClusterAlerts)
+	alerts, err := client.GetClusterAlerts(alertCtx, types.GetClusterAlertsRequest{
+		Labels: labels,
+	})
+	if err != nil && !trace.IsNotImplemented(err) {
+		trace.Wrap(err)
+	}
+
+	types.SortClusterAlerts(alerts)
+	var errs []error
+	for _, alert := range alerts {
+		if err := alert.CheckMessage(); err != nil {
+			errs = append(errs, trace.Errorf("Skipping invalid alert %q: %v", alert.Metadata.Name, err))
```

Comment:
```suggestion
			errs = append(errs, trace.Errorf("invalid alert %q: %w", alert.Metadata.Name, err))
```

Errors should be lowercase and terse, as they might get composed (just like we're doing here).